File: src/main/java/org/lendi/umtapo/entity/Library.java
package org.lendi.umtapo.entity;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;


/**
 * Created by axel on 29/11/16.
 */
@Entity
public class Library {

 @Id
 @GeneratedValue(strategy = GenerationType.AUTO)
 private Integer id;
 private String name;
 private Integer shelfMarkNb;
 private Boolean useDeweyClassification;
 private Boolean subscriptionDuration;
 private Integer borrowDuration;
 private String currency;


 public Integer getId() {
  return id;
 }


 public void setId(Integer id) {
  this.id = id;
 }


 public String getName() {
  return name;
 }


 public void setName(String name) {
  this.name = name;
 }


 public Integer getShelfMarkNb() {
  return shelfMarkNb;
 }


 public void setShelfMarkNb(Integer shelfMarkNb) {
  this.shelfMarkNb = shelfMarkNb;
 }


 public Boolean getUseDeweyClassification() {
  return useDeweyClassification;
 }


 public void setUseDeweyClassification(Boolean useDeweyClassification) {
  this.useDeweyClassification = useDeweyClassification;
 }


 public Boolean getSubscriptionDuration() {
  return subscriptionDuration;
 }


 public void setSubscriptionDuration(Boolean subscriptionDuration) {
  this.subscriptionDuration = subscriptionDuration;
 }


 public Integer getBorrowDuration() {
  return borrowDuration;
 }


 public void setBorrowDuration(Integer borrowDuration) {
  this.borrowDuration = borrowDuration;
 }


 public String getCurrency() {
  return currency;
 }


 public void setCurrency(String currency) {
  this.currency = currency;
 }
}

File: front/src/entity/library.ts
export class Library {
  id: number;
  name: string;
  shelfMarkNb: number;
  useDeweyClassification: boolean;
  subscriptionDuration: number;
  borrowDuration: number;
  currency: string;
  defaultZ3950: number;

  getId(): number {
    return this.id;
  }

  getName(): string {
    return this.name;
  }

  setName(value: string) {
    this.name = value;
  }

  getShelfMarkNb(): number {
    return this.shelfMarkNb;
  }

  setShelfMarkNb(value: number) {
    this.shelfMarkNb = value;
  }

  getUseDeweyClassification(): boolean {
    return this.useDeweyClassification;
  }

  setUseDeweyClassification(value: boolean) {
    this.useDeweyClassification = value;
  }

  getSubscriptionDuration(): number {
    return this.subscriptionDuration;
  }

  setSubscriptionDuration(value: number) {
    this.subscriptionDuration = value;
  }

  getBorrowDuration(): number {
    return this.borrowDuration;
  }

  setBorrowDuration(value: number) {
    this.borrowDuration = value;
  }

  getCurrency(): string {
    return this.currency;
  }

  setCurrency(value: string) {
    this.currency = value;
  }

  getDefaultZ3950(): number {
    return this.defaultZ3950;
  }

  setDefaultZ3950(value: number) {
    this.defaultZ3950 = value;
  }
}
